pkg/hsskey/hsskey.go

```diff
@@ -33,7 +34,9 @@ type blobReader interface {
 }
 
 const (
-	hostSecretSeedLen = 32
+	hostSecretSeedLen        = 32
+	hostSecretSeedStructSize = 64
```

Comment:
How would you describe the difference between these two ?

Reply:
hostSecretSeedStructSize is the total length of serialized HSS struct
hostSecretSeedLen is the length of the seed value we use to unlock disk